zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -164,8 +165,11 @@ public Boolean call() {
                 logError(e, "Exception saving collection after approval exception").collectionName(collection).log();
             }
 
-            SlackNotification.alarm(String.format("Exception approving collection %s : %s",
-                    collection.getDescription().getName(), e.getMessage()));
+            SlackNotification.collectionAlarm(collection,
+                    "Exception approving collection",
+                    new PostMessageField("Error", e.getMessage(), false)
+            );
```

Comment:
Should this notification be inside the above catch block? Unless I am reading it incorrectly as it is it will always send this notification.

Reply:
It's inside the catch block, the inner catch block is handling cases where we set the approval status to error and fail to save - either way I'd assume we want to notify in slack?